If we have Wine, it will provide cmd so we can build the pkg-config path
for the test.
---
 check/check-path   | 2 +-
 check/config.sh.in | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/check/check-path b/check/check-path
index 37244d9..2a6be75 100755
--- a/check/check-path
+++ b/check/check-path
@@ -22,7 +22,7 @@ if [ "$native_win32" = yes ]; then
         # Assume we have cmd somewhere to get variable %cd%. Make sure
         # to strip carriage returns.
         pcdir=$(cd $top_builddir/.libs &&
-                cmd /C echo %cd% | sed -r 's/\r//g')
+                $WINE cmd /C echo %cd% | sed -r 's/\r//g')
     fi
     win_path="$pcdir/lib/pkgconfig;$pcdir/share/pkgconfig"
 
diff --git a/check/config.sh.in b/check/config.sh.in
index 68fa097..8b7ffb5 100644
--- a/check/config.sh.in
+++ b/check/config.sh.in
@@ -15,3 +15,4 @@ system_library_path="@system_library_path@"
 list_indirect_deps=@use_indirect_deps@
 PACKAGE_VERSION=@PACKAGE_VERSION@
 native_win32=@native_win32@
+WINE=@WINE@
-- 1.8.1.4 